### DISCUSSION TITLE
[Feature] add turkish to supported languages

### DISCUSSION THREAD

**Discussion by ysf3rdm** [immich-app/immich#12727] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

add turkish to supported languages 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by danieldietzler** [comment]
We already support turkish?

**Reply by danieldietzler** [reply]
<img width="779" alt="Screenshot 2024-09-16 at 16 26 47" src="https://github.com/user-attachments/assets/4e40cc8f-4470-44e5-a450-0a65602e7105">
<img width="1275" alt="Screenshot 2024-09-16 at 16 26 58" src="https://github.com/user-attachments/assets/e55b6e23-6e07-4a84-a6f0-9a2ff3b2cadb">

**Reply by ysf3rdm** [reply]
there is no turkish selection in the andorid app
![image](https://github.com/user-attachments/assets/a9ae3ada-16d3-4de1-a7f5-69856622b421)

**Reply by danieldietzler** [reply]
Ah sorry, you had ticked all boxes initially, so I got confused. We plan on migrating mobile to weblate soon-ish, at which point turkish will also come :)

**Reply by ysf3rdm** [reply]
ah my bad, I didn't check the web also, okay thanks.

**Reply by ysf3rdm** [reply]
if needed i can add turkish translate to help you guys

**Reply by danieldietzler** [reply]
It's currently only at 53%, so more translations on [weblate](https://hosted.weblate.org/projects/immich/immich) would be appreciated there :)
